### DISCUSSION TITLE
seasons..?

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#84] [Ideas]
should the game have seasons? seasons could limit what can be grown at what time, and maybe also what is for sale at the shop. it doesn't have to be anything in depth, but since there are "days" implemented in the game already, it could naturally make sense to add a layer of seasons on top of this.

just for discussions sake, let's say every 30 days changes the seasons. alternatively, since time moves fast in this game and there is no forced idling, we could implement a layer of months as well. and have a few months = a season.

**Comment by jeremyckahn** [comment]
I am very into this idea! It's pretty significant so it would require quite a bit of planning. I think it would be worth spinning up a separate project board for this to define and track the the subtasks. 

Before we can reasonably start on this, we'd need quite a few more crops because currently there are only 10 and we'd only want 25-50% available for a given season. We'd also need to figure out what should happen during Winter since normal crops can't typically grow then.

**Reply by lstebner** [reply]
great points, also glad to hear you're into the idea! maybe we can start a project board for new crops to start? since i imagine there are several steps involved in creating each one. winter calls for greenhouse(s).... yeah, lots of work!

**Reply by jeremyckahn** [reply]
That makes sense to me! I've started a new board at https://github.com/jeremyckahn/farmhand/projects/3.

Feel free to add your ideas as cards there and convert them to Issues as you see fit!

As a side note, I think this feature would be significant enough to warrant a `major` version bump.

**Comment by karlerik-simon** [comment]
I really like this idea!
